lib/ethon/easy/http/actionable.rb

```diff
@@ -70,7 +70,12 @@ def form
         # @param [ easy ] easy the easy to setup.
         def setup(easy)
           @easy = easy
+
           if params.empty?
+            # This is called here to have the side effect of removing
+            # the :array_encoding key from options, to avoid forwarding
+            # it to Curl later. FIXME.
+            array_encoding
```

Comment:
Sure, I'll gladly refactor the Actionable initialization.

`params_encoding` looks good to me. Even though it only affects arrays, for now, it's more explanatory and forward-looking.